HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -251,6 +251,99 @@ def customizeHLTfor43025(process):
     return process
 
 
+def customizeHLTfor43257(process):
```

Comment:
In case this PR still needs to be finalised, this customisation (which is not the usual PR-specific customisation to avoid breaking the HLT menus in the release) could be moved to a separate file in this same directory, named `customizeHLTforAlpaka.py` [*]. I understood this is the strategy we plan to follow to test the Alpaka HLT customisations before they are integrated in the HLT menu itself (via ConfDB). @fwyzard and @cms-sw/hlt-l2 can correct me, of course.

[1] Something like the following under `HLTrigger/Configuration/python/customizeHLTforAlpaka.py`
```py
import FWCore.ParameterSet.Config as cms
[..]

def customizeHLTforAlpakaECALLocalReco(process):
      [..]
      return process

def customizeHLTforAlpaka(process):
      process = customizeHLTforAlpakaECALLocalReco(process)
      [..]
      return process

Reply:
@missirol thanks for the suggestion.
@thomreis can you make this change ?

Reply:
>   can you make this change ?

wasn't it done already in the last push?

Reply:
and ... yes, I just realised that I was looking at an outdated diff :-/